### ISSUE TITLE
Network differences

### ISSUE THREAD

**Issue by YAYA530** [kevinmcgregor/mdine#10]
Hi Kevin.

I found the code in GitHub only tell us how to plot a network for each group plot_networks (obj). Is there any code for plotting the differential Network between case and control samples (like figure 5 in your paper)? And I found the command [sig_diff_prec.Rd], is it for finding the significantly different correlation between case and control groups? What about the weight for the differential network?

**Comment by kevinmcgregor** [comment]
Yes there is: check out https://github.com/kevinmcgregor/crohns_analysis for how to make the figures from the paper.

Assume your mdine object is `md.fit`

Estimated differences in weights are in `md.fit$post_mean$invsigma_diff`

Credible intervals for differences in correlation are found in `md.fit$ci$invsigma_diff` . These can be used to determine significance.
